lib/symlink.go

```diff
@@ -135,7 +135,23 @@ func ChangeProductSymlink(product Product, binVersionPath string, userBinPath st
 		err = CreateSymlink(binVersionPath, location)
 		if err == nil {
 			logger.Debugf("Symlink created at %q", location)
-			logger.Warnf("Run `export PATH=\"$PATH:%s\"` to append %q to $PATH", dirPath, location)
+
+			var isDirInPath bool = false
+			checkDirPath := strings.TrimRight(dirPath, "/")
+
+			for _, path := range strings.Split(os.Getenv("PATH"), ":") {
```

Comment:
We have lots of location, path etc. variables floating around - is it worth denoting this as envVariablePath or similar?

Reply:
Yeah, agree, it makes sense. Done via aeabba5
Please take a look again. Thanks.

Reply:
Thanks for valuable suggestions. Will be looking into improving #551 now 👍🏻